tests/blackbox/data/evaluation/namespace-scoped/bitnami_no_rbac_permissions/deployment.yml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: bitnami-no-rbac-permissions
+---
+# ServiceAccount that can be used by the pod to access cluster resources
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pod-reader-sa
+  namespace: bitnami-no-rbac-permissions
+---
+# RoleBinding that binds the Role to the ServiceAccount
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: read-pods-binding
+  namespace: bitnami-no-rbac-permissions
+subjects:
+- kind: ServiceAccount
+  name: pod-reader-sa
+  namespace: bitnami-no-rbac-permissions
+roleRef:
+  kind: Role
+  name: pod-reader
+  apiGroup: rbac.authorization.k8s.io
+---
```

Comment:
what happens if you don't create these rbac resources and don't set any service account to it? 
Will we still get the rbac error?

Reply:
this is about a case where Role is missing.

Reply:
maybe I should rename it to `bitnami-role-missing`?

Reply:
this would also make sense, if we want to extend this scenario, with RoleBinding missing, or Role exists, but no permissions were given

Reply:
ok, please rename it then.

Reply:
done